template.yaml

```
@@ -48,4 +48,76 @@ Resources:
         - Key: !Sub "tag:${TagName}"
           Values:
             - !Ref TagValue
-      AutomationTargetParameterName: InstanceId
\ No newline at end of file
+      AutomationTargetParameterName: InstanceId
+
+  RequireDeleteTag:
```

Ich wuerde evtl.
```suggestion
  RequireManagedTag:
```
vorschlagen, der tag ist ja quasi frei konfigurierbar